sockeye/data_io.py

```diff
@@ -1260,7 +1260,7 @@ def parallel_iterate(source_iterators: Sequence[Iterator[Optional[Any]]],
         "Different number of lines in source(s) and target iterables.")
 
 
-class FileListReader(Iterator):
+class FileListReader:
```

Comment:
did you get typing errors or why did you remove this?

Reply:
Yes, newer versions of pylint were raising a warning:
```
E0239: Inheriting 'Iterator', which is not a class. (inherit-non-class)
```